### ISSUE TITLE
onMonthChanged event

### ISSUE THREAD

**Issue by jmercado86** [FadyFayezYounan/easy_date_timeline#36]
How can I get the event for when the Month Changes?

**Comment by sagnik-sanyal** [comment]
I guess for now you can store the date in a local variable and compare the months with the onDayChanged callback, whenever it changes compare that month of your cached variable is equal to the current day month or not, if not update it and call setState to update wherever you want in your ui.

**Comment by jmercado86** [comment]
I don't think it would work because the onDateChange event doesn't trigger when you change the Month on the header and instead it only triggers when you tap on a Day. Please correct me if I'm wrong. Thank you

**Comment by sagnik-sanyal** [comment]
Ok yeah I got your question, yes now it will only fire if you tapped on the date. Hmm ok then I have an idea that you have to change the internal implementation and check whether the date in the current scroll offset has changed months and add a callback like onMonthChanged and fire that when required.

**Comment by d9media** [comment]
I would be interested in a scroll event for EasyInfiniteDateTimelineController too. It would be good to have the currently focused event before it's clicked.
